CMakeLists.txt

```diff
@@ -427,9 +431,9 @@ if (DEPS_DIR)
     get_cmake_property(cacheVariableNames CACHE_VARIABLES)
     foreach(cacheVariableName ${cacheVariableNames})
         set(cacheVariableValue "${${cacheVariableName}}")
-        if (cacheVariableValue MATCHES "/(linux64|macosx64|mingw32|mingw64|msvc32|msvc64)-([0-9]+)/")
-            if (NOT ${CMAKE_MATCH_2} EQUAL ${DEPS_VERSION})
-                message(WARNING "You seem to be using a dependency from version ${CMAKE_MATCH_2}"
+        if (cacheVariableValue MATCHES "/(linux|windows|macos)-(i686|amd64)-(default,mingw,msvc)_([0-9]+)/")
```

Comment:
You need to detect the old naming scheme too.

Reply:
That should be fixed.